doc/src/reference/py.rst

```diff
@@ -167,7 +222,11 @@ program
     :type params: dict[str, argument]
 
     :return: The result of the last instruction.
-    :rtype: argument
+    :rtype: list[argument]
+
+.. py:method:: sort()
+
+    Sort the modules of the program such that instructions appear in correct occurrence.
```

Comment:
Maybe its better to say this does a topological sort.

Reply:
Done.